### ISSUE TITLE
HELP: Can you help me with a script?

### ISSUE THREAD

**Issue by gerroon** [zadam/trilium#774]
Hi

I know nothing about js at all so I am wondering if someone is willing to modify the existing date script so it does what I am looking for?

I would like another `today` button/script that creates a note named `DD-MM-YYYY` under the currently selected note? 

The current `today` script works good for journaling fashion but I am looking for that modified version.

**Comment by gerroon** [comment]
Is this possible at all?

**Comment by zadam** [comment]
> Is this possible at all?

Yes. You can use `api.getActiveTabNote()` in frontend API and then use that as a parent in backend's `api.createNewNote()`.

**Comment by gerroon** [comment]
Thanks but I know rabiit poop about js :( It would be nice if someone can give me the full script with the necessary css I guess.

thanks
